constants.py

```diff
@@ -0,0 +1,50 @@
+import discord
+HELP_REPLY = discord.Embed(title="Ferris Elf help page", color=0xE84611, description="""
```

Comment:
COMMENT: I think it would be better to add fields here, instead of dumping all commands in the description box, this would ease readability